### ISSUE TITLE
about attribute information

### ISSUE THREAD

**Issue by majun19970125** [Hsintien-Ng/Adma-GAN#1]
Hi, can you share the attribute information of the CUB dataset

**Comment by Hsintien-Ng** [comment]
Yes, the attribute information is similar with the attribute label in CUB dataset.
We have uploaded the corresponding .txt file.

**Comment by majun19970125** [comment]
Hi, your code is missing utils and attributes_captions_embedding.pickle files, can you make them public?

**Comment by Hsintien-Ng** [comment]
Sorry, I was busy before and didn't pay attention to this repo. I have now uploaded the correponding files.
Sorry again for the missing files because the code has been reorganized.

**Comment by Ashutosh1995** [comment]
I have requested access @Hsintien-Ng  for the dataset. Could you please approve it?

**Comment by Hsintien-Ng** [comment]
@Ashutosh1995 The preprocessed CUB dataset is available at https://drive.google.com/file/d/1SDVUxKbfFCp9gkYsXqzTa_fl-hFk68eK/view
which is introduced in the readme.md

**Comment by Ashutosh1995** [comment]
![Snip20230209_2](https://user-images.githubusercontent.com/11408850/217736755-0f860bb0-4a1b-48fe-bd0c-cf237b78b52e.png)

It says that the request has been sent. There is no option of directly downloading the datasets from the link.

**Comment by Hsintien-Ng** [comment]
@Ashutosh1995 Can you get it now?

**Comment by Ashutosh1995** [comment]
Yes, now it is accessible. Thanks, @Hsintien-Ng!

**Comment by lcxsnow** [comment]
Hi, could you also share the attribute information of the MSCOCO dataset? @Hsintien-Ng
